src/services/item/plugins/app/appData/service.ts

```diff
@@ -85,7 +101,10 @@ export class AppDataService {
       },
     );
 
-    return appDataRepository.post(itemId, actorId, completeData);
+    return appDataRepository.post(itemId, actorId, completeData).then(async (newAppData) => {
+      await this.hooks.runPostHooks('post', member, repositories, { appData: newAppData, itemId });
+      return newAppData;
+    });
```

Comment:
Written like this we expect to not wait for the posthook. I know we don't need to wait for the websockets, but I would add the `await` to be consistent with other parts of the code. What do you think @dialexo ?